doc/classes/ProjectSettings.xml

```diff
@@ -2890,11 +2890,11 @@
 			If [code]true[/code], forces vertex shading for all rendering. This can increase performance a lot, but also reduces quality immensely. Can be used to optimize performance on low-end mobile devices.
 		</member>
 		<member name="rendering/textures/canvas_textures/default_texture_filter" type="int" setter="" getter="" default="1">
-			The default texture filtering mode to use on [CanvasItem]s.
+			The default texture filtering mode to use for [CanvasItem]s built-in texture. In shaders, this texture is accessed as [code]TEXTURE[/code].
 			[b]Note:[/b] For pixel art aesthetics, see also [member rendering/2d/snap/snap_2d_vertices_to_pixel] and [member rendering/2d/snap/snap_2d_transforms_to_pixel].
 		</member>
 		<member name="rendering/textures/canvas_textures/default_texture_repeat" type="int" setter="" getter="" default="0">
-			The default texture repeating mode to use on [CanvasItem]s.
+			The default texture repeating mode to use for [CanvasItem]s built-in texture. In shaders, this texture is accessed as [code]TEXTURE[/code].
```

Comment:
```suggestion
			The default texture repeating mode used to render the built-in texture for [CanvasItem] nodes. In shaders, this texture is accessed as [code]TEXTURE[/code].
```

Reply:
I don't think this addition is correct. Or, at the very least, it is confusing. 

This setting changes how the texture is sampled when it is used in a shader. In some situations that may change how the CanvasItem is rendered to the Viewport, and in others it won't. Importantly though, this setting doesn't change how the texture itself is created, which this addition seems to say. 

Sampling is the process of reading from a texture. So basically this tells the shader what to do when it is asked to read from an area outside the texture's boundary.